tests/unit_tests/scheduler/test_job.py

```diff
@@ -110,22 +110,17 @@ async def test_job_run_sends_expected_events(
     realization: Realization,
     monkeypatch,
 ):
-    monkeypatch.setattr(
-        ert.scheduler.job,
-        "forward_model_ok",
-        lambda _: LoadResult(forward_model_ok_result, ""),
-    )
+    async def load_result(_):
+        return (forward_model_ok_result, "")
+
+    monkeypatch.setattr(ert.scheduler.job, "forward_model_ok", load_result)
     scheduler = create_scheduler()
     monkeypatch.setattr(
         scheduler.driver,
         "read_stdout_and_stderr_files",
         lambda *args: "",
     )
 
-    scheduler.job.forward_model_ok = MagicMock()
-    scheduler.job.forward_model_ok.return_value = LoadResult(
-        forward_model_ok_result, ""
-    )
```

Comment:
These lines did nothing, they manipulated the wrong version of the object, so thus deleted.